app/src/lib/stores/github-user-store.ts

```diff
@@ -185,6 +182,7 @@ export class GitHubUserStore extends BaseStore {
       .sort(
         (x, y) => compare(x.ix, y.ix) || compare(x.user.login, y.user.login)
       )
+      .slice(0, maxHits)
```

Comment:
yay! I think this is way better than the previous logic, which could discard items that would have appeared at the top of the list 😀